configs/development-1.env.json

```diff
@@ -38,6 +38,7 @@
   },
   "streamrUrl": "http://127.0.0.1",
   "streamrAddress": "0xFCAd0B19bB29D4674531d6f115237E16AfCE377c",
+  "storageNodeRegistry": null,
```

Comment:
Going with the vision that storage is its own thing, I will close this point. Since there will be no need for the storage node to know about other storage nodes.